src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -138,30 +114,32 @@ export default Backbone.Model.extend({
   setDefaultFilterStates() {
     this.set({ filters: {}, states: this.getDefaultSelectedStates() });
   },
-  getSelectedStates() {
-    const availableStateFilterIds = this.getAvailableStates().map('id');
-    const selectedStates = this.getStatesFilters();
-
-    return filter(selectedStates, id => contains(availableStateFilterIds, id)).join() || NIL_UUID;
-  },
   getFiltersState() {
     return {
       filters: this.getFilters(),
       states: this.getStatesFilters(),
       defaultStates: this.getDefaultSelectedStates(),
     };
   },
-  getEntityFilter() {
```

Comment:
This function was creating the filters for every worklist app every time instead of just the one that this state needs.  I also broke this up by filter-type